platform/kube/inject/initializer_test.go

```diff
@@ -96,7 +96,8 @@ func TestInitialize(t *testing.T) {
 		in                     string
 		wantPatchBytesFilename string
 		policy                 InjectionPolicy
-		managedNamespace       string
+		includeNamespaces      []string
+		excludeNamespaces      []string
```

Comment:
I did not see a test which covers following:

includeNamepaces is `ibm-system`, but deployment is using `google-system` as namespace and this kind of applications should also be ignored by the initializer.

The code logic already covered this, but no test case, good to ship this one and we can follow up to add more test cases in other PRs.

Thanks @ayj !

Reply:
That should be handled [here](https://github.com/ayj/pilot/blob/a1874a335c6c8655c11dde8ee1f0f7e8bd4a993f/platform/kube/inject/initializer_test.go#L190) given `s/foo/google-system` and `s/bar/ibm-system`.